mixer/adapter/prometheus/server_test.go

```diff
@@ -32,6 +32,16 @@ func TestServer(t *testing.T) {
 		t.Fatalf("Start() failed unexpectedly: %v", err)
 	}
 
+	s3 := newServer(testAddr)
+
+	if err := s3.Start(test.NewEnv(t), http.HandlerFunc(doesNothing)); err != nil {
+		t.Fatalf("Start() failed unexpectedly: %v", err)
+	}
+
+	if s.Port() != 9992 {
```

Comment:
should we be hardcoding the port?  can we just find an open port and use that?  perhaps by opening a listener on `:0`, saving the port number, and then closing it before returning?

Reply:
Yes. Fixed